.github/workflows/build-and-deploy.yml

```diff
@@ -88,6 +88,41 @@ jobs:
           maven-central-password: ${{ secrets.MAVEN_CENTRAL_PASSWORD }}
           signing-key: ${{ secrets.SIGNING_KEY }}
           signing-password: ${{ secrets.SIGNING_PASSWORD }}
+  # Test jpackage utility
+  test-packaging:
+    needs:
+      - select-java-version
+    strategy:
+      matrix:
+        os: [ windows-2022, macos-12, ubuntu-22.04 ]
+    runs-on: ${{ matrix.os }}
+    if: >-
+      !github.event.repository.fork
+      && (
+        github.event_name != 'pull_request'
+        || github.event.pull_request.head.repo.full_name == github.repository
+      )
+    concurrency:
+      group: ${{ github.workflow }}-test-packaging-${{ matrix.os }}-${{ github.event.number || github.ref }}
+      cancel-in-progress: true
+    steps:
+      - name: Checkout
+        uses: danysk/action-checkout@0.2.14
+      - name: Setup Java
+        uses: actions/setup-java@v4.0.0
+        with:
+          java-version: ${{ needs.select-java-version.outputs.java-version }}
+          distribution: 'temurin'
+      - name: Dry run
+        shell: bash
+        run: ./gradlew jpackage --info -Djpackage.dryRun=true
+      - name: Install bsdtar
+        if: matrix.os == 'ubuntu-22.04'
+        run: sudo apt update && sudo apt install libarchive-tools
```

Comment:
Is there a container that can do this instead?

Reply:
Something like [this](https://hub.docker.com/r/tim03/libarchive)? 
Unfortunately github removed libarchive-tools from the ubuntu vm and I did not find any alternative tool to extract a rpm package. I tried also rpm2cpio | cpio, but it was way slower and did not work from gradle.